rules/prefer-node-protocol.js

```diff
@@ -32,8 +32,8 @@ const create = () => ({
 		if (
 			typeof value !== 'string'
 			|| value.startsWith('node:')
+			|| /^bun(:\w+|$)/.test(value)
 			|| !isBuiltinModule(value)
-			|| !isBuiltinModule(`node:${value}`)
```

Comment:
Why are we removing this line?

Reply:
no longer needed, as it was only added to detect bun's modules

Reply:
I don't understand what's the problem then, are you saying both `bun:foo` and `node:bun:foo` passes `isBuiltinModule()` check?

Reply:
`isBuiltinModule(bun:foo)` returns `true` when eslint is run on bun - https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2383#issue-2355755215

Reply:
<details>

<summary>Screenshot</summary>

![image](https://github.com/sindresorhus/eslint-plugin-unicorn/assets/115353812/73bbb574-826c-4998-bbe0-6bc2741c8c77)

</details>

Reply:
https://github.com/sindresorhus/is-builtin-module/blob/2b22980d890cd656419106e2d2cca5696453bd7f/index.js#L12-L14